src/patterns/gen-ai/aws-langchain-common-layer/index.ts

```diff
@@ -49,16 +58,16 @@ export interface LangchainLayerProps {
 }
 
 /**
-   * @summary The LangchainCommonDepsLayer class.
```

Comment:
can we keep the original name for those 2 constructs as discussed in the separate PR ?